docs/upgrade/upgrade_zonemaster_backend_ver_8.0.0.md

```diff
@@ -90,12 +98,17 @@ than v8.0.0, and not upgraded, use the following instructions.
 Run
 ```sh
 cd $(perl -MFile::ShareDir -le 'print File::ShareDir::dist_dir("Zonemaster-Backend")')
-perl patch_sqlite_db_zonemaster_backend_ver_8.0.0.pl
+perl patch/patch_sqlite_db_zonemaster_backend_ver_8.0.0.pl
 ```
 
 ### MySQL (or MariaDB)
 
-Run
+First update the privileges of the `zonemaste` user:
+```sh
+sudo mysql -e "GRANT ALL ON zonemaster.* TO 'zonemaster'@'localhost';"
+```
+
+then run
 ```sh
 cd $(perl -MFile::ShareDir -le 'print File::ShareDir::dist_dir("Zonemaster-Backend")')
```

Comment:
Also her there is `$(...)` that does not work with FreeBSD. Please use backtick instead. The same on 8 lines down.

Reply:
fixed